### ISSUE TITLE
Aborted (core dumped)

### ISSUE THREAD

**Issue by vqbang** [AlexeyAB/darknet#3801]
Hi @AlexeyAB,

After loading the yolov2 weights from https://pjreddie.com/media/files/yolov2.weights, I was able to train about 2000 iterations using this command line: 
./darknet detector train data/obj.data ~/yolov2-obj.cfg ~/yolov2.weights -dont_show -mjpeg_port 5000 -map
It started to train at iteration 500200 (pretrained weights yolov2.weights), but after iteration 5020xx, the program crashed and it showed the following result:
![image](https://user-images.githubusercontent.com/49437404/63508686-9f05a880-c504-11e9-8da8-d14479b3da9f.png)
![image](https://user-images.githubusercontent.com/49437404/63498245-b2f2df80-c4ef-11e9-82a5-b8da6f2b0607.png)

Can you help me find out what's happening? Many thanks.

**Comment by beHappy666** [comment]
the same problem, have you fixed it?

**Comment by vqbang** [comment]
This issue happened only when I tried to turn on the CUDNN and CUDNN_HALF OR monitored the result through web server using the command line that I had mentioned above as I remember. I figured out that I was using the tensorflow 1.12.3 with CUDA 9.0 and after installing tensorflow 1.14 with CUDA 10 and a correspond CUDNN, the problem was solved.

**Comment by AlexeyAB** [comment]
@vqbang So did you solve the problem?

**Comment by beHappy666** [comment]
Maybe wrong label in your own data.

**Comment by vqbang** [comment]
> @vqbang So did you solve the problem?

Yes, the problem is solved ^^

**Comment by vqbang** [comment]
> Maybe wrong label in your own data.

I think it isn't, because I'd trained the model for about 2000 iterations (~4 epochs) and was able to plot the result to web, but I think it's related to the CUDNN.
